packages/angular_devkit/build_angular/src/tools/sass/rebasing-importer.ts

```diff
@@ -74,6 +83,26 @@ abstract class UrlRebasingImporter implements Importer<'sync'> {
       updatedContents.update(start, end, rebasedUrl);
     }
 
+    for (const { start, end, value } of findImports(contents)) {
+      // Currently only provide directory information for known/common packages:
+      // * `@material/`
+      // * `@angular/`
+      // Comprehensive pre-resolution support may be added in the future.
+      if (!value.startsWith('@angular/') && !value.startsWith('@material/')) {
```

Comment:
Question: Any particular reason for limiting to `@angular/` and `@material/`? Just staying safe in case it breaks something?

Reply:
Safer for now and avoids some more complex logic for determining if it is relative or an actual module reference.  CSS/Sass don't require `./` or `../` for relative paths. So for general support, a bare specifier would need to be checked for relative path viability first. Technically a project could have a relative `@angular` directory but that's unlikely and easy to check/remedy (add `./` to the specifier).

Reply:
Makes sense. Might be useful to include some of that in the comment so we know the constraints if/when we change or open up the package restriction.